tests/test_modeling_common.py

```diff
@@ -539,7 +539,10 @@ def check_hidden_states_output(inputs_dict, config, model_class):
                 outputs = model(**self._prepare_for_class(inputs_dict, model_class))
             hidden_states = outputs[-1]
 
-            self.assertEqual(len(hidden_states), self.model_tester.num_hidden_layers + 1)
+            expected_num_layers = getattr(
+                self.model_tester, "expected_num_hidden_layers", self.model_tester.num_hidden_layers + 1
```

Comment:
Added this part because the full funnel transformer actually adds two hidden states, not just one.

Reply:
One for encoder / one for decoder or is this something else?

Reply:
Ah yeah got it - nvm